db/db_impl/db_impl.h

```diff
@@ -1240,6 +1240,38 @@ class DBImpl : public DB {
 
   std::atomic<bool> shutting_down_;
 
+  class RecoveryVersionEdits {
+   public:
+    ~RecoveryVersionEdits() {
+      for (size_t i = 0; i < edit_lists.size(); i++) {
+        for (size_t j = 0; j < edit_lists[i].size(); j++) {
+          delete edit_lists[i][j];
+        }
+        edit_lists[i].clear();
+      }
+      cfds.clear();
+      mutable_cf_opts.clear();
+      edit_lists.clear();
+    }
+
+    void UpdateVersionEdits(ColumnFamilyData* cfd, VersionEdit& _edit) {
+      if (map.find(cfd->GetID()) == map.end()) {
+        uint32_t size = static_cast<uint32_t>(map.size());
+        map.emplace(cfd->GetID(), size);
+        cfds.emplace_back(cfd);
+        mutable_cf_opts.emplace_back(cfd->GetLatestMutableCFOptions());
+        edit_lists.emplace_back(autovector<VersionEdit*>());
+      }
+      uint32_t i = map[cfd->GetID()];
+      edit_lists[i].emplace_back(new VersionEdit(_edit));
+    }
+
+    std::unordered_map<uint32_t, uint32_t> map;  // cf_id to index;
+    autovector<ColumnFamilyData*> cfds;
+    autovector<const MutableCFOptions*> mutable_cf_opts;
+    autovector<autovector<VersionEdit*>> edit_lists;
```

Comment:
We should follow https://google.github.io/styleguide/cppguide.html#Variable_Names when naming the variables.